plugins/module_utils/identity/bitwarden.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.common.text.converters import to_text
+
+
+class BitwardenException(Exception):
+    pass
+
+
+class Client(object):
+    '''A wrapper for for bw command.'''
+
+    # Path to the `bw` executable.
+    cli_path = 'bw'
+    # AnsibleModule using the client.
+    module = None
+
+    @staticmethod
+    def run(args, stdin=None, expected_rc=0):
+        '''Run the bw client.
+
+        Args:
+            args(list): Arguments to pass to the bw CLI client.
+            stdin(str): Data to write to the client's stdin.
+            expected_rc(int): Expected return code of bw CLI client.
+
+        Return:
+            (str, str): The contents of standard out and standard error.
+        '''
+        rc, out, err = Client.module.run_command([Client.cli_path] + args, data=stdin, binary_data=True)
```

Comment:
It is a good practice to search for the command's full path before running it:
```suggestion
        rc, out, err = Client.module.run_command([Client.module.get_bin_path(Client.cli_path, required=True)] + args, data=stdin, binary_data=True)
```